docs/catalog/cloud/remove-unused-assets.md

```diff
@@ -0,0 +1,42 @@
+---
+version: 1.0
+submitted_by: markus-ntt-seidl
+published_date: TBD
+category: cloud
+tags: 
+ - cloud
+ - role:cloud-engineer
+ - size:small
+---
+
+# Remove unused assets
+
+## Description
+
+Monitor and analyze the application and the cloud bill to identify resources that are no longer used or can be reduced.
+
+## Solution
+
+Decommission unused assets to free consumed resources. Consolidate resources, for example different database instances with a only a few tables could be moved into one instance. Review generated assets if they contain redundant content and handle them accordingly. 
+Inspect storage buckets if unused assets can be removed.
+
+## SCI Impact
+
+`SCI = (E * I) + M per R`  
+[Software Carbon Intensity Spec](https://grnsft.org/sci)
+
+With respect to the SCI equation, remove unused assets will impact one part:
+
+- `E`: Reduces the embodied emissions by not using the resources
+
+## Assumptions
+
+- Assumes that resources can be decommissioned
+
+## Considerations
+
+- Before decommissioning resources consider workloads that run rarely, for example generating quarterly or yearly reports
```

Comment:
I'm not sure what this means or is referring to?

Reply:
This pattern is about removing unused assets. Unused is determined by the user of the cloud account and is not done by a computer. I assume that one would look at a log and see, this haven't been used for 2 weeks -> can be shut down. That would be harmful for, maybe, monthly database cleanup jobs which runs every 4weeks. (just an example).

This is following the rule, that every pattern must have an assumption / consideration to pass SME review.

Reply:
"This is following the rule, that every pattern must have an assumption / consideration to pass SME review" -- I think that is a general guideline, not a hardened rule. 

That said, maybe it's a little clearer to say you can consider prioritizing decommissioning resources that that infrequently run? I think the wording here doesn't have an action with it other than to consider which makes it unclear what the user is supposed to be considering about it.